[clapack] Fix linux build

kafeg opened the pull request:
Fix for https://github.com/Microsoft/vcpkg/issues/5368

PhoebeHui commented:
@kafeg, thanks for contributing to vcpkg! 

Here is fail triplet(x64-windows-static) and regression(openblas) with your PR.

Log:
 Processing x64-windows-static                   757 vs 767
```
    blaze                                                        Skip vs Fail
     ceres                                                       Skip vs Pass
     clapack                                                     Skip vs Pass
     dlib                                                        Skip vs Pass
     ensmallen                                                   Skip vs Pass
     g2o                                                         Skip vs Pass
     geogram                                                     Skip vs Pass
     mlpack                                                      Skip vs Pass
     openblas                      **regression**                Fail vs Pass
     openmvg                                                     Skip vs Pass
     sophus                                                      Skip vs Fail
     suitesparse                                                 Skip vs Pass
```

seanwarren commented on ports/openblas/portfile.cmake:
```diff
@@ -16,9 +16,11 @@ if(NOT VCPKG_TARGET_ARCHITECTURE STREQUAL "x64")
-if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
-    message("openblas currenly only supports dynamic library linkage")
-    set(VCPKG_LIBRARY_LINKAGE "dynamic")
+if(VCPKG_CMAKE_SYSTEM_NAME  STREQUAL "WindowsStore")
```
This check will only trigger on UWP builds, not normal windows builds (as is currently required to build `OpenBLAS` successfully). I would suggest the following change which matches the `OpenBLAS` restriction

```suggestion
if(MSVC)
```

PhoebeHui commented:
@seanwarren, could you also update CONTROL revision in https://github.com/Microsoft/vcpkg/blob/master/ports/clapack/CONTROL and 
https://github.com/Microsoft/vcpkg/blob/master/ports/openblas/CONTROL? 

More information in https://github.com/Microsoft/vcpkg/blob/master/docs/maintainers/control-files.md

seanwarren commented:
@PhoebeHui - this isn't my PR so I don't have write access (I just made a code review suggestion earlier). I think @kafeg will need to update the control file. 

Cheers

vicroms commented:
It is failing due to a post-build validation:

```
DLLs should not be present in a static build, but the following DLLs were found:

    C:/vsts/_work/1/s/packages/openblas_x64-windows-static/bin/openblas.dll
    C:/vsts/_work/1/s/packages/openblas_x64-windows-static/debug/bin/openblas.dll

There should be no bin\ directory in a static build, but ...\packages\openblas_x64-windows-static\bin is present.
There should be no debug\bin\ directory in a static build, but ...\packages\openblas_x64-windows-static\debug\bin is present.
If the creation of bin\ and/or debug\bin\ cannot be disabled, use this in the portfile to remove them

    if(VCPKG_LIBRARY_LINKAGE STREQUAL static)
        file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
    endif()

Found 2 error(s). Please correct the portfile:
```

cenit committed:
[tiff] use vcpkg wrapper for all configs, remove dead code (#5584)

* [tiff] use vcpkg wrapper for all configs, remove dead code

* [tiff] update CONTROL

emmatyping committed:
Add nuget to vcpkg_find_acquire_program (#4568)

* Add nuget to vcpkg_find_acquire_program

* Add niget to list of programs one can download

JackBoosY committed:
[libssh]fix config.cmake error, add usage. (#5666)

LarryIII committed:
[luasocket] Fix static build for renaming files (#5665)

Commit:
[qt5] Fix mac build (#5624)

* [qt5] Fix mac build

* [qt5] Disable parallel build on mac

* [qt5] Fix qmake find on mac

* [qt5] Update find jom for mac

* [qt5] Fix invocation of build command

* [qt5-base] Rebuild packages for #5624

* [harfbuzz] Remove disabled assert (failing due to -Werror)

* [qt5] Bump control version

* [qt5] Disable harfbuzz on mac

* [qt5-base] Disable harfbuzz on mac

rollebolle committed:
Angle updated to master, UWP fix for Angle (#4862)

* Angle updated to master, UWP fix for Angle

* updated uwp patch

RT2Code committed:
[physx] Initial port (#5618)

* [physx] Initial port

* [physx] use vcpkg version of cmake if not installed

myd7349 committed:
[roaring] Add new port (#5643)

* [roaring] Add new port

* [roaring] Add a warning for 32-bit build

grdowns committed:
Merge pull request #5634 from edmBernard/fix_realsense2

[realsense2] fix cmake for linux

donny-dont committed:
Add headers to vcpkg_download_distfile (#5640)

* Add headers to vcpkg_download_distfile

Allows headers to be appended when downloading a distribution. This allows for setting
authentication headers to download from a private repository.

* [scripts,docs] Regenerate docs for portfile functions

* [docs,scripts] Fix typo regressions

* [docs,scripts] Fix broken link

cenit committed:
[darknet/yolo] new port (#5648)

* [darknet/yolo] add draft implementation

* [darknet] use official repo

* [darknet] use stb from vcpkg

* [darknet] Explicit failure on ARM and UWP

Neumann-A committed:
fix missing debug references in cmake generated (release) target files.

Neumann-A committed:
cmake sometimes requires a generator if it is not defined beforehand.
extra code comments. Maybe the test should actually try to link targets from the package instead  of just testing find_package

Neumann-A committed:
default szip to also build static libraries in shared builds

Neumann-A committed:
update szip control

Neumann-A committed:
updated hdf5 to 1.10.5.
Also now builds dynamic and static libs in dynamic build due to the targets exported by hdf5. (Revert to default hdf5 build behavior)

Neumann-A committed:
kealib update to 1.4.11 and correctly include hdf5

Neumann-A committed:
update netcdf-c tp 4.6.2 and fixed curl linkage

Neumann-A committed:
fixed flann for linux

Neumann-A committed:
fix highfive in static builds.
HDF5_USE_STATIC_LIBRARIES must be set in static builds so that find_package(HDF5) finds the static hdf5 libraries

Neumann-A committed:
fixes #5562

Neumann-A committed:
correct hdf5 usage in netcdf-cxx4

Neumann-A committed:
workaround for qhull regression in static build see #5574

Neumann-A committed:
vtk update to 8.2.0

Neumann-A committed:
fix find_package for fftw3 and add arch features (#4783)

* fix find_package for fftw3

* the patch took care of moving the files

* Additional Patch to add Suffix to LibraryDepends

* fixed minsizerel and relwithdebinfo for vs cmake configs

* remove fix for minsizerel and relwithdeb info. Use global cmake mappings of these targets instead!

* added feature packages. unfortunally avx and avx2 create an ICE with Visual Studio 2017 (maybe fixed with 2019?)

* removed sse2 as default (forgot about arm...)

khizmax committed:
* [libcds] Update to v2.3.3 (#5274) (#5278)

* * [libcds] Update to v2.3.3 (#5274)

 * Minimum CMake version is 3.0.2
 * Changed software license from BSD to Boost
 * Removed '#pragma lib' for Windows build
 * Bugfix and minor improvements

* [libcds] Use PATCHES parameter instead of `vcpkg_apply_patches()`

* [libcds] Fix static builds

ras0219-msft committed:
[vcpkg] Fix infinite loop bug on certain filesystem implementations when searching up

JackBoosY committed:
[curl]Add the dependency of the curl feature [ssh] to solve the lnk2019 error. (#5682)

tobyc11 committed:
[openexr] Fix IlmImf.dll installed to the wrong location (#5677)

Rastaban committed:
Merge pull request #5574 from Neumann-A/VTK_8.2.0

[VTK 8.2.0/HDF5 1.10.5] Upgrade VTK and HDF5

ceeac committed:
[capstone] Update Capstone to v4.0.1-49-ge3d5a594 (#5546)

* Update Capstone to v4.0.1-49-ge3d5a594

* Use full SHA hash for ref

* Build Capstone with dynamic runtime

* Update Capstone to 4.0.1-57-g120373dc

* Fix wrong description for osxkernel feature

tzneal committed:
[portaudio]: fix Linux build (#5678)

* [portaudio]: fix Linux build

Wrap some Win specific stuff in an if (WIN32)

Fixes #4526

* [portaudio] Bump CONTROL version

Rastaban committed:
Merge pull request #5679 from ras0219-msft/dev/roschuma/fix-recursive-find-up

[vcpkg] Fix infinite loop bug on certain filesystem implementations when searching up

angelmixu committed:
[fontconfig] use __attribute__((visibility("default"))) for exporting symbols on macOS (#5689)

* use __attribute__((visibility("default"))) for exporting symbols on macOS

* [fontconfig] Bump CONTROL version

NancyLi1013 committed:
[cairo] Update version to 1.16.0 (#5646)

BigSeb committed:
[sqlpp11] Moved the python scripts located in the bin folder to a separate folder. This removes the warning when compiling in static.

BigSeb committed:
[sqlpp11-connector-mysql] Removed the include files in the debug/include directory which was preventing compilation of this package.

BigSeb committed:
[sqlpp11-connector-sqlite3] Removed the include files in the debug/include directory which was preventing compilation of this package.

xgcssch committed:
[upstream] Update toversion 2.8.81 and added missing folder  (#5668)

* Update to upstream version 2.8.81
Copy import directory to the share folder, so it can be used by clients.

myd7349 committed:
[libxml2] Update to 2.9.9 (#5193)

* [libxml2] Update to 2.9.9

Fix #5177

* [libxml2] Fix static build

* [libxml2] Modernize

* [libxml2] Use vcpkg_from_github

* [libxml2] Get version info from configure.ac

* [libxml2] Install xmlversion.h

* [libxml2] Improve port file

* [libxml2] Move version.h generation into CMakeLists.txt

* [libxml2] Fix misspelled version header

* [libxml2] Add UWP support

grdowns committed:
Merge pull request #5577 from cwfitzgerald/spirv-tools-target

[spirv-tools] Add cmake targets

xgcssch committed:
[gsoap] update to 2.8.82 (#5695)

cwfitzgerald committed:
[glslang] Properly fixup cmake targets

ras0219-msft committed:
[harfbuzz] Restore patches removed by #5359 (#5696)

Commit:
[fmt] Changed library linkage to static. (#5688)

* [fmt] Changed library linkage to static.

* [fmt] Re-enabled shared build without breaking LTCG toolchains.

kiwixz committed:
update to cmake 3.14 (allow VS2019RC)

kiwixz committed:
fix vs2019 arch specification

cwfitzgerald committed:
[cppfs] Fix debug exports (#5708)

cwfitzgerald committed:
[mapbox-variant] Fix include install location due to hard-coded paths (#5706)

* [mapbox-variant] Fix include install location due to hard-coded paths

edmBernard committed:
[orc] update to 1.5.5 (#4786)

* [orc] update to 1.5.3

* [orc] update usage with zstd

* [orc] update to 1.5.4

* [orc] update to orc 1.5.5rc1

* [orc] fix static build

* [orc] bump to 1.5.5

MouhammadEV committed:
[WIP][vtk] add the option -DModule_vtkGUISupportMFC=ON (#4661)

* Update portfile.cmake

* [vtk] Only enable Module_vtkGUISupportMFC for Windows. Require atlmfc on Windows.

grdowns committed:
Merge branch 'master' of https://github.com/microsoft/vcpkg into dev/grdowns/5699

grdowns committed:
Update version string

grdowns committed:
Merge pull request #5699 from cwfitzgerald/glslang-fixup

[glslang] Properly fixup cmake targets

grdowns committed:
Merge branch 'master' of https://github.com/microsoft/vcpkg into dev/grdowns/5692

myd7349 committed:
[qcustomplot] Add new port (#5036)

* [qcustomplot] Add new port

* [qt5-base] Trigger a rebuild for qcustomplot

* Revert "[qt5-base] Trigger a rebuild for qcustomplot"

This reverts commit 1082892d12958a207435310098587306d0f3114b.

* [qcustomplot] Trigger rebuild on CI

grdowns committed:
Update version strings

cwfitzgerald committed:
[rapidxml-ns] Add new port. (#5703)

* [rapidxml-ns] Add new port

* [rapidxml] Move headers out of include to prevent conflicts and clutter.

* [rapidxml-ns] Update SHA512

* [rapidxml-ns] Fix header installation

* [3fd] Fix header issue with rapidxml

grdowns committed:
Merge pull request #5692 from BigSeb/fix-sqlpp11

Fix sqlpp11 warnings

Commit:
[gli]Fix gli is incompatible with glm

jwillemsen committed:
Make use of vcpkg_copy_pdbs instead of manually copying the pdb files (#5733)

Squareys committed:
[magnum] Update patches (#5730)

Signed-off-by: Squareys <squareys@googlemail.com>

xgcssch committed:
[gsoap] Include dom.* files and plugin directory (#5735)

Rastaban committed:
[qpid-proton] add libuv on osx

use lubuv on osx as the proactor.

Rastaban committed:
[collada-dom] use vcpkg minizip/zip.h

it is possible to find the wrong zip.h when doing a full CI test.  Force it to
find the right one.

grdowns committed:
Merge pull request #5726 from LarryIII/FixUncombination

[gli]Fix gli is incompatible with glm

Rastaban committed:
[rxqt] use fixed release for rxqt

ras0219-msft committed:
[rapidjson] Fix --head support (#3693)

ras0219-msft committed:
[vcpkg] Update to CMake 3.14.0

LarryIII committed:
[fribidi] Update Version to support static build (#5745)

BillyONeal committed:
Update cpprestsdk to v2.10.11. (#5748)

JackBoosY committed:
[clapack]fix missing library and tools. (#5749)

ras0219-msft committed:
Merge remote-tracking branch 'origin/master' into HEAD

ras0219-msft committed:
[vcpkg] Revert change to freebsd cmake version

ras0219-msft committed:
[vcpkg] Fixup previous commit

ras0219-msft committed:
[vcpkg] Fixup previous commit

Rastaban committed:
[ismrmrd] make optional hdf5 dependency explicit (#5757)

* [ismrmrd] make optional hdf5 dependency explicit

This port will behave differently in the CI build depending on if hdf5 was installed
before it.  Make the hdf5 dependency explicit so it will behave consistently.

Rastaban committed:
[liblemon][lemon] deprecate duplicate port, forward to original (#5758)

Commit:
[cpr] Changed library linkage to static. (#5705)

* [cpr] Changed library linkage to static.

* [cpr] Removed unintentional file.

* [cpr] Cleanup and bump version

grdowns committed:
Revert "Use feature macro to check for support of filesystem"

grdowns committed:
Merge pull request #5765 from Microsoft/revert-5651-check-for-filesystem

Revert "Use feature macro to check for support of filesystem"

Commit:
[tiff] Changed library linkage to static. (#5766)

cenit committed:
[freeopcua] better dealing with dynamic builds (#5763)

* [freeopcua] better dealing with dynamic builds

SuperWig committed:
Add new port cli (#5762)

JackBoosY committed:
[draco]Fix build error on x86-uwp. (#5755)

Ambrou committed:
[kdsoap] add package (#5750)

* create kd-saop port

Committer: PETITGENET AMBROISE <apetitgenet@free.fr>

* [kd-soap] remove extra comments

NancyLi1013 committed:
[grpc] Update grpc to 1.9.1. (#5744)

mauricek committed:
[qt5-mqtt] new port Qtmqtt (#5736) (#5737)

* [qt5-modularscripts] Split build steps

When building modules not available from the vendor download side, the
build process stays the same. Create two functions, one for accessing
the sources, the other for compilation.

* [qtmqtt] Add new port (#5736)

soroshsabz committed:
[rttr] Update 0.9.6. (fixes #5710) (#5712)

* Update rttr library to 0.9.6. (fixes #5710)

* Add log message for not supported platform. (see #5710)

cenit committed:
[darknet] fix static build (#5771)

cenit committed:
[open62541] new port

cenit committed:
[open62541] remove unnecessary patch, regex in portfile works also with head rev

tarcila committed:
Disable cmake integration test for now

Integration test is failing for now because of #5630 and #5635
This at least makes the package function on Windows x86 and x64.

vicroms committed:
Disable vcpkg_test_cmake()

Commit:
[giflib] Changed library linkage to static.

yurybura committed:
[boost-python] Restore unwind-type.patch

wravery committed:
[cppgraphqlgen] Update to 2.1.0 (#5797)

SuperWig committed:
[doctest] Update to 2.3.1 (#5792)

luncliff committed:
[coroutine] arch, commit (#5787)

* version: 1.4.1
* check architecture before vcpkg_from_github

x86 error message contains repository path to prevent confusing

Signed-off-by: luncliff <luncliff@gmail.com>

jwillemsen committed:
[ace] Add support for vcpkg install ace on Linux (#5802)

* [ace] Add support for vcpkg install ace on Linux

    * ports/ace/CONTROL:
    * ports/ace/portfile.cmake:

* Fix mwc logfile name

    * ports/ace/portfile.cmake:

* Fixed windows detection

    * ports/ace/portfile.cmake:

yurybura committed:
[boost-python] Restore unwind-type.patch (#5795)

rfric committed:
[pbc] Correct compilation warnings (#5794)

Wataru343 committed:
[freeimage]Fix link symbols of tiff. (#5786)

Commit:
[angle] Fixed build with /permissive- toolchain on Windows and clang on Linux (#5785)

JackBoosY committed:
[libnice]Fix build error in windows. (#5799)

* [libnice]Fix build error in windows.

* [libnice] switched to vcpkg_extract_source_archive_ex

JackBoosY committed:
[apr-util]Fix static library usage bug. (#5769)

* [apr-util]Fix static library usage bug.

* [apr-util]Fix macro error.

LilyWangL committed:
[nlohmann-json]update version to 3.6.0 (#5746)

JackBoosY committed:
[openblas]Fix white-space-path configure bug. (#5741)

JackBoosY committed:
[libmikmod]fix missing dll bug. (#5743)

* [libmikmod]fix missing dll bug.

* [libmikmod]1.Set macro to replace fixed library suffixes to support multiple platforms   2.Copying dynamic library files is disabled when building static libraries

* [libmikmod]1. change vcpkg_apply_patches to vcpkg_extract_source_archive_ex.   2.delete commented code.

* [libmikmod] remove extra 'set'

colatkinson committed:
Update to 0.1.15, and update CMakeLists.txt

colatkinson committed:
Remove unnecessary patch to libnice

cenit committed:
[open62541] add system dependencies for linux, quickly error-out for UWP

past-due committed:
[nlohmann-json] update version to 3.6.1

yurybura committed:
[python3] Update to v3.7.3

Helco committed:
[SDL2_xxx]: Add CMake targets for SDL2 addons (#5754)

* Add SDL2 image target

* Add SDL2_mixer target

* Add SDL2_ttf target

* Add SDL2_net target

* Fix missing source file for SDL2_image

* Disable harfbuzz for freetype properly

* Use both include dirs for freetype in SDL2-tff

zhihaoy committed:
[tbb] Upgrade to 2019_U4 (#5760)

* [tbb] Upgrade to 2019_U4

This release fixes a bug that causes problem in vTune.

* [tbb] whitespace change to force rebuild in CI

Commit:
Add Qt5 static missing dependencies, fix mac packages (#5680)

* [qt5-macextras] Add qt5-macextras port

* [cutelyst2] Fix install on mac

* [qt5-purchasing] Add missing dependency on qt5-declarative

* [qt5] Ensure we link to required libraries for qt5 static build

* [qscintilla] Fix install on mac

* [libqglviewer] Fix install paths on mac

* [libqglviewer] Fix regressions

* [qscintilla] Fix regressions

* Bump version numbers

* [qt5] restore harfbuzz on osx

past-due committed:
[physfs] Update to version 3.0.2 (#5821)

past-due committed:
[SDL2] Add patch for SDL2 hid_enumerate bug 4391 (#5820)

Applies patch for SDL 2.0.9 bug 4391: "hid_enumerate() sometimes causes game to freeze for a few seconds"

See: https://bugzilla.libsdl.org/show_bug.cgi?id=4391

Rastaban committed:
[nlohmann-json] update version to 3.6.1 (#5818)

frabert committed:
[sf2cute] new port (#5814)

* add sf2cute port

* clean up sf2cute

* fix sf2cute project version

* Fix sf2cute port

* Use latest upstream tag

Rastaban committed:
Merge pull request #5822 from yurybura/master

[python3] Update to version 3.7.3

Rastaban committed:
Merge pull request #5617 from tarcila/master

[capnproto] Add Capnproto package

Rastaban committed:
Merge pull request #5779 from cenit/dev/cenit/open62541

[open62541] new port

ras0219-msft committed:
[giflib] Modernize. Avoid use of WINDOWS_EXPORT_ALL_SYMBOLS.

Rastaban committed:
Merge pull request #5811 from colatkinson/libnice-0.1.15

Update libnice to 0.1.15

Rastaban committed:
Merge pull request #5784 from qis/dev/qis/giflib

[giflib] Changed library linkage to static.

ewertons committed:
[azure-iot] vcpkg update for master/public-preview release (#5789)

* [azure-iot] vcpkg update for master/public-preview release

* [azure] Fix download hash strings

* [azure] Bump CONTROL versions

LilyWangL committed:
[many ports]Add long path warnings

eao197 committed:
[json-dto] updated to v.0.2.8 (#5836)

* sobjectizer updated to v.5.5.24.3

* json-dto updated to v.0.2.8

lazyhamster committed:
[libmspack] update to 0.10.1. (#5834)

Rastaban committed:
Merge pull request #5835 from wangli28/dev/Lily/LongPathWarning

[many ports]Add long path warnings

JackBoosY committed:
[mozjpeg]Fix installation files to the wrong directory (#5833)

* [jemalloc]Fix dll generated when generating static library

* Revert "[jemalloc]Fix dll generated when generating static library"

This reverts commit a833bf5593be181e065af097732c78cb810888fd.

* [mozjpeg]1.Fix installation files to the wrong directory.   2.Change how dynamic and static library built macros are used

BillyONeal committed:
Update Casablanca to v2.10.12. (#5830)

Commit:
[gdal] fix for building static library (#5839)

JackBoosY committed:
[jemalloc]Fix dll generated when generating static library (#5829)

* [jemalloc]Fix dll generated when generating static library

LarryIII committed:
[pcre2/caffe2] Fix space issue (#5827)

* [pcre2/caffe2] Fix space issue

CJ-Johnson committed:
Removes string replacement on Abseil headers

Abseil now has all uses of min and max wrapped in parens to prevent macro expansion. This will continue to be the case going forward, so these replacements are no longer needed.

Also, the Github web UI is forcing an extra change that updates the return character to the proper Windows CR+LF

donny-dont committed:
Allow additional environment variables to be set in a clean environment (#5650)

* Allow additional environment variables to be set in a clean environment

* Add documentation for VCPKG_KEEP_ENV_VARS

Rastaban committed:
[decimal-for-cpp] update to 1.16 (#5852)

cenit committed:
[libhungarian] add port (#5849)

* [libhungarian] add port

* [hungarian] fix cmake targets

myd7349 committed:
[liblzma] Fix for UWP build (#5607)

JackBoosY committed:
[giflib]fix compile error in x64-uwp (#5578)

* [giflib]fix compile error in x64-uwp

* [giflib]Modify the location of the CMakelists.txt changes in the patch to avoid patching failures.

* [giflib]Modify cmakelists.txt directly and delete the corresponding changes in the patch

* [giflib] Modernize. Avoid use of WINDOWS_EXPORT_ALL_SYMBOLS.

myd7349 committed:
[protobuf] Update to 3.7.1 (#5502)

* [protobuf] Update to 3.7.1

Commit:
[exiv2] Update exiv2 to 0.27 release, optionally support unicode on windows (#5454)

Rastaban committed:
[freetype] fix issue #5774 (#5853)

* load 0005-Fix-DLL-EXPORTS.patch

* Create 0005-Fix-DLL-EXPORTS.patch

* [freetype] cleanup whitespace

zhihaoy committed:
[ampl-mp] Limit the scope of ICE workaround (#5870)

JackBoosY committed:
[signalrclient]Fix build error (#5868)

* [signalrclient]Fix build error: LNK2019 with openssl

NancyLi1013 committed:
[jsonnet] Fix white-space-path issue. (#5862)

JackBoosY committed:
[luajit]move luajit headers to the directory "include/luajit/" to avoid file conflicts. (#5863)

[luajit]move luajit headers to the directory "include/luajit/" to avoid file conflicts.

JackBoosY committed:
[abseil]Fix link error when using abseil library. (#5860)

* [jemalloc]Fix dll generated when generating static library

* Revert "[jemalloc]Fix dll generated when generating static library"

This reverts commit a833bf5593be181e065af097732c78cb810888fd.

* [abseil]Fix link error when using abseil library.

Rastaban committed:
Removes string replacement on Abseil headers (#5850)

Abseil now has all uses of min and max wrapped in parens to prevent macro expansion. This will continue to be the case going forward, so these replacements are no longer needed.

Also, the Github web UI is forcing an extra change that updates the return character to the proper Windows CR+LF

JackBoosY committed:
[pdal]fix missing vendor headers (#5846)

* [pdal]1.fix missing vendor headers.   2.modify the patch name and add the serial number.

Commit:
[glew] Do not build utilities (#5859)

* [glew] Do not build utilities

Commit:
[opencv] Make opencv build exportable (#5857)

* [opencv] Make opencv build exportable

* [opencv] bump version number

Commit:
[xerces-c] Add libraries required on osx to target using wrapper (#5858)

* [xerces-c] Add libraries required on osx to target using wrapper

* [xerces-c] Bump version number

JackBoosY committed:
[libharu]fix build failure. (#5628)

* [libharu]fix build failure.

* [libharu]Comment more code

myd7349 committed:
[brunocodutra-metal] Add new port (#5817)

* [brunocodutra-metal] Add new port

* [brunocodutra-metal] Rename

Commit:
Fix for shared build broken due to #5839 (#5882)

greg7mdp committed:
add parallel-hashmap package (#5893)

greg7mdp committed:
[parallel hashmap] increment release version (#5898)

* update version of package

Helco committed:
[libuv/uvw] Update to 1.27.0 (#5895)

* Update libuv to 1.27.0

* Update uvw to 1.15.0

markdegreef committed:
[fcl] fix linux build (#5582)

* [fcl] fix linux build

Set correct path of cmake config files

markdegreef committed:
[octomap] fix linux build (#5583)

* [octomap] fix linux build

Copy entire bin folder for linux, as it only contains the executables

* [octomap] remove debug binaries

Asozial committed:
gdal: fixed typo (#5904)

* gdal: fixed typo

* Update CONTROL file

SuperWig committed:
[effolkronium-random] Update to 1.3.0 (#5900)

MistGebaut committed:
sdl1: static build (#5903)

* sdl1: static build

* Update CONTROL file

cbezault committed:
Check scripts/cmake/* and buildsystems/vcpkg.cmake for changes when hashing

kiwixz committed:
[libwebp] do not include all feature by default (#5886)

* [libwebp] do not include all feature by default

* [libwep] bump control version

ShelleyLake committed:
[osg] static build (#5913)

lrineau committed:
Update CGAL to version 4.14 (#5734)

* CGAL now depends also on Boost Multiprecision
* Update to release 4.14

dlech committed:
Fix vcpkg_find_acquire_program(NUGET) (#5871)

Set the ARCHIVE and NOEXTRACT variables so that downloading works. Also HASH is case-sensitive, so make it lower case.

vcpkg_find_acquire_program(NUGET) was failing with the following error message:

-- Downloading https://dist.nuget.org/win-x86-commandline/v4.8.1/nuget.exe...
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:159 (file):
  file DOWNLOAD error: cannot create directory
  'C:/Users/david/Documents/GitHub/vcpkg/downloads/temp/C:/Users/david/Documents/GitHub/vcpkg/downloads'
  - Specify file by full path name and verify that you have directory
  creation and file write privileges.
Call Stack (most recent call first):
  scripts/cmake/vcpkg_find_acquire_program.cmake:255 (vcpkg_download_distfile)
  ports/gobject-introspection/portfile.cmake:20 (vcpkg_find_acquire_program)
  scripts/ports.cmake:71 (include)

vicroms committed:
Merge branch 'master' of https://github.com/Microsoft/vcpkg into pr/cmake_3_14

vicroms committed:
[sqlitecpp] Find sqlite3 includes from vcpkg

vicroms committed:
[libodb-sqlite] Find sqlite3 includes from vcpkg

vicroms committed:
[libmodbus] Remove duplicated header files

vicroms committed:
[darknet,openssl-windows,tbb] Rebuild on CI

dzenanz committed:
Fix double-conversion issue in ITK (#5115)

* Using newer version of ITK which fixes double-conversion problem

* Minor style update of ITK portfile

* Using newer version of ITK which fixes double-conversion problem

* [itk] Improve version string

* [itk][hdf5] Update ITK commit. Depend on eigen3. Bump hdf5 control version.

* [hdf5] Update CONTROL

* [netcdf-c] Disable parallel configure due to writes to source directory

* [hdf5] Disable parallel configure due to writes to source directory. Switch to tar.gz to prevent CRLF line endings.

ras0219-msft committed:
[vcpkg-fetch] Set git subpath to mingw32\bin\git.exe to fix execution within containers (#5878)

ras0219-msft committed:
[vtk] Fix flakiness by relying on vcpkg-provided dependencies
[libharu] Add patches required for vtk
[vcpkg-build-cmake] Add retry logic for mt.exe failures

ras0219-msft committed:
[hdf5][vtk] Add vcpkg-cmake-wrapper for HDF5 to fix vtk

Cheney-W committed:
[live555] Update the download path (#5919)

Rastaban committed:
[openssl-windows] partially back out moderinzation change

LilyWangL committed:
[openmesh]Update version to 8.0 (#5923)

Commit:
[Thrift] Updates 2019.03.29 (#5924)

angelmixu committed:
update muParser to v.2.2.6.1 (#5931)

Cheney-W committed:
[milerius-sfml-imgui] Fix find_package issue (#5930)

Commit:
[RocksDB] Updates 2019.03.29 (#5933)

LarryIII committed:
[date] Fix failures in UWP with latest commit (#5932)

grdowns committed:
Merge branch 'dev/roschuma/vtk' of https://github.com/ras0219-msft/vcpkg into dev/grdowns/5935

grdowns committed:
[hdf5] Set wrapper policy for older dependent libraries

grdowns committed:
Merge branch 'pr/cmake_3_14' of https://github.com/kiwixz/vcpkg into dev/grdowns/5702

djarek committed:
Fix installation failure when system Boost present (#5939)

Boost 1.70 introduces "native" cmake config scripts which are
accidentally used when installing a vcpkg package. This only occurs
when Boost 1.70 is installed in the system when the user uses
`vcpkg install`.

Signed-off-by: Damian Jarek <damian.jarek93@gmail.com>

Rastaban committed:
[qca] don't use port dir for temp (#5954)

* [qca] don't use port dir for temp

* [qca] bump control version

philjdf committed:
[arrow] Update Arrow to v0.13 (#5950)

* Update arrow to 0.13.0

* dynamic build working

* Both builds now work

* modifications to work on Linux

* Turn of jemalloc

zharkov-eu committed:
Update msgpack to 3.1.1 (#5945)

Neumann-A committed:
[fftw3] fix git patch (#5926)

* Update fftw3_arch_fix.patch

fix fftw3 arch fix

* bump version in fftw3 control

Commit:
[fix][ffmpeg] fixing linux build (#5917)

* limit windows dep libs to windows

* fixing configure patch

* fixing detect-openssl.patch

* adapting portfile to linux

* adding a linux-specific build sh

* add missing endif()

* bumping version

* adding extra check for windows environment

fradav committed:
highfive bump to 2.0 and macos fix (same as win32) (#5955)

* highfive bump to 2.0 and macos fix (same as win32)

* macos fix

vicroms committed:
[gdal] Apply conditional patching (#5940)

grdowns committed:
[folly] Link against appropriate lib for build type

Rastaban committed:
[glew][octomap][roaring] fix parallel config (#5958)

* [glew][octomap][roaring] fix parallel config

fix CMake Error: Could not open file for write in copy operation

* [anax] fix parallel config

cenit committed:
[giflib] enable ninja (#5963)

koprok committed:
Fix LLVM cmake modules (#5848)

* [llvm] Reduce memory usage during linking

* [llvm] Fix cmake modules path

grdowns committed:
Merge pull request #5935 from ras0219-msft/dev/roschuma/vtk

[vtk] Fix flaky builds by using system libraries

grdowns committed:
Merge branch 'master' of https://github.com/microsoft/vcpkg into dev/grdowns/5702

Rastaban committed:
[freeimage] fixup dependencies (#5969)

coryan committed:
Update google-cloud-cpp to v0.8.0. (#5972)

kiwixz committed:
[ebml,matroska] update ebml to 1.3.7, fix installation on linux (#5981)

* [ebml] fix installation on linux

* [ebml] update to 1.3.7

* [ebml] fix config installation on linux

* [matroska] cleanup, fix linux compilation

rickertm committed:
Add port for solid3 (#5989)

pantor committed:
[inja] Update to v2.1.0 (#5985)

* Update CONTROL

* Update portfile.cmake

david-antiteum committed:
[opentracing] v1.5.1 (#5974)

* OpenTracing #5970

* No UWP. Static requires dynamic #5970
- Dynamic version can be build without the static one
- Static version can only be build when dynamic is also active
- UWP is not supported

* Linux support #5970

jwillemsen committed:
[ace] Upgrade to 6.5.5 (#5992)

* ports/ace/CONTROL:
    * ports/ace/portfile.cmake:

koprok committed:
[pixman] Fix build on x86-linux (#5998)

grdowns committed:
Merge pull request #5702 from kiwixz/pr/cmake_3_14

Update to cmake 3.14 (allow compilation with VS2019RC v142)

jwillemsen committed:
[ace] Add support for Visual Studio 2019 (#6001)

* ports/ace/CONTROL:
    * ports/ace/portfile.cmake:

xiao-ai committed:
update sqlite3 to 3.27.2 (#6002)

Commit:
[Tbb] Updates 2019.03.29 (#5928)

* [Tbb] Updates 2019.03.29

* Set the tbb  installed libs in TBBConfig.cmake

NancyLi1013 committed:
[g2o] Fix g2o:x64-windows failed with POST_BUILD_CHECKS_FAILED.#4098 (#5925)

* [g2o] fix g20:x64-windows failed with POST_BUILD_CHECKS_FAILED.#4098

* Add "VCPKG_USE_HEAD_VERSION" to judge if it is from HEAD.

ras0219-msft committed:
[angle] Fix regressions introduced in #4862: (#5922)

Reintroduces the unofficial:: targets.
PREFER_NINJA.
Small simplifications/modernizations.

ras0219-msft committed:
[vcpkg] Synchronize vcpkg-base with external repo (#5934)

past-due committed:
[curl] Fix `tool` feature (#5971)

- Use appropriate EXECUTABLE_SUFFIX to support non-Windows platforms
- Fixup `curl-target-release.cmake` to point to the new executable location (fixing the CURL::curl target)

past-due committed:
[google-cloud-cpp] Fix libcurl linkage (#5968)

* [google-cloud-cpp] Fix libcurl linkage

* [curl] Fix libcurl exported CMake target config (Windows)

edmBernard committed:
[soci] bump version (#5964)

cenit committed:
[darknet] add compatibility with OpenCV 4 (#6018)

past-due committed:
[curl] Bump CONTROL version (#6012)

LarryIII committed:
[sdl1]Fix sdl1 conflict with sdl2 (#6011)

xnorpx committed:
Use Opus CMake #4875 (#6007)

cbezault committed:
Print what port installed confliciting files

cbezault committed:
Merge branch 'master' into master

vicroms committed:
Revert "Print what port installed confliciting files"

This reverts commit 79682317c9e087dc1d1ae450e3edd03b0fa0cc59.

ras0219-msft committed:
Complete revert of accidental merge b1b7ec5c0be3a3b0

alexander-smyslov committed:
[activemq-cpp] Add activemq-cpp library (#6017)

Commit:
[fix][ffmpeg] changing $PATH separator for Linux (#6015)

* [fix][ffmpeg] changing $PATH separator for Linux

Using a semicolon in $PATH might break on some flavors of Linux, eg Docker based on Ubuntu 18.04.

tony979 committed:
Enable openssl:arm64-windows

Rastaban committed:
partial update for testing

Rastaban committed:
add git hash

Rastaban committed:
Revert "add git hash"

This reverts commit 1a3b388a911f844e6907cc9058728e2a3bdda419.

Rastaban committed:
Revert "partial update for testing"

This reverts commit 880181892e83b89657207777b1f2ce40fe3d31cb.

grdowns committed:
Merge pull request #6030 from tony979/openssl-arm64

Enable openssl:arm64-windows (Fix #4478, #4812)

grdowns committed:
[openssl-windows] Bump version number

grdowns committed:
Merge pull request #6044 from Microsoft/dev/grdowns/bump-openssl-windows-ver

[openssl-windows] Bump version number

Cheney-W committed:
[openvdb] Fix openvdb[tools] build failures (#6028)

soroshsabz committed:
Add -allowAppleClang option to bootstrap.sh. (fixes #4698) (#5915)

* Add -allowAppleClang option to bootstrap.sh. (fixes #4698)

* [vcpkg-bootstrap] Simplify appleclang changes and reformat

* [vcpkg-bootstrap] Remove enviroment variable VCPKG_ALLOW_APPLE_CLANG

atkawa7 committed:
[distorm, scylla_wrapper, devicenameresolver] init (#4170)

* distorm init

* scylla wrapper init

* devicenameresolver init

* [devicenameresolver][distorm][scylla-wrapper] Cleanup and mark libraries as dll-only.

Rastaban committed:
[devil/libcopp/spaceland] stable cmake config (#6046)

cenit committed:
[darknet] sync with community expected version, add fixes for downstream projects (#6047)

cbezault committed:
Print what port installed confliciting files (#6037)

grdowns committed:
Merge branch 'master' of https://github.com/kafeg/vcpkg into dev/grdowns/5525

grdowns committed:
Bump version numbers, fix system name check

grdowns commented:
Thanks for the contributions, everyone!
